content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,150 @@
 ---
-title: "Working with Kubernetes Objects"
-weight: 40
+title: Objects in Kubernetes
```

Comment:
What is the "record of intent" for moving the text from "working-with-objects/kubernetes-objects.md" to "working-with-objects/_index.md" ?

A side effect I can perceive is the new redirect entry which implies an additional round trip between my browser and the web server.

Reply:
Is https://github.com/kubernetes/website/pull/31261/commits/997db9a88d7caabf4be0dccc84a63f4de8058060 the answer you were looking for @tengqm ?

I was worried that https://kubernetes.io/docs/concepts/overview/working-with-objects/ isn't very approachable for a newcomer, so I opted to move the overview page one level up.

I could add an extra commit to find pages that link to the old URI and change them to reference the new one.  Should I do that?

Reply:
I think the existing page is very approachable. We have a shortcut on the "front page".

<img width="827" alt="image" src="https://user-images.githubusercontent.com/5094756/183394846-639a8ede-3b69-45e0-9424-12cd2b45b56b.png">

Reply:
I was imagining that some readers might arrive at https://kubernetes.io/docs/concepts/overview/working-with-objects/ by working through the left-hand nav of the site structure, with each page shown in order.

Reply:
Right. If we want to improve the index page, why don't we just replace the generated indice by a tweaked list of links. If we change this `_index.md` file as proposed in this PR, users will likely to have difficulties finding the entries to topics like "name and IDs", "annotations", "field and selectors" etc. Those pages are deeply nested already and they are not shown on the left navigation.

If you want to improve the object concept page, please focus you change to the `working_with_objects/kubernets_objects.md` page rather than moving it elsewhere. If you can convince me that this relocation is a good move, please propose it in a separate PR.

/hold